src/utils/non-existing-path.ts

```diff
@@ -0,0 +1,23 @@
+import * as vscode from 'vscode';
+
+/**
+ * Finds the file, which is not exists in filesystem.
```

Comment:
```suggestion
 * Finds the file, which does not exist in filesystem.
```

 but does it really finds, if it does not exist?